### PR TITLE
[HWORKS-746] Documentation for custom commands and python environment history

### PR DESCRIPTION
**Pull request by gibchikafa**

### CODE REVIEW

File: docs/user_guides/projects/python/environment_history.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Python Environment History
+The Hopsworks installation ships with a Miniconda environment that comes preinstalled with the most popular libraries you can find in a data scientist toolkit, including TensorFlow, PyTorch and sci-kit-learn. The environment may be managed using the Hopsworks Python service to install or libraries which may then be used in Jupyter or the Jobs service in the platform.
```
Comment (o-alex) [inline docs/user_guides/projects/python/environment_history.md:2]: Second sentence is missing words:
`to install or libraries`

File: docs/user_guides/projects/python/environment_history.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+The Python virtual environment is shared by different members of the project. When a member of the project introduces a change to the environment i.e., installs/uninstalls a library, a new environment is created and it becomes a defacto environment for everyone in the project. It is therefore important to track how the environment has been changing over time i.e., what libraries were installed, uninstalled, upgraded, or downgraded when the environment was created and who introduced the changes. 
+
+In this guide, you will learn how to run how you can track the changes of your Python environment.
```
Comment (o-alex) [inline docs/user_guides/projects/python/environment_history.md:6]: extra words:
`how to run`

File: docs/user_guides/projects/python/environment_history.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Viewing python environment history in the UI
+The Python environment evolves over time as libraries are installed, uninstalled, upgraded, and downgraded. To assist in tracking the changes in the environment, you can see the environment history in the UI. You can view what changes were introduced at each point a new environment was created. Hopsworks will keep a version of a YAML file for each environment so that if you want to restore an older environment you can use it. To see the differences between environments click on the button as shown in figure 1. You will then see the difference between the environment and the previous environment it was created from.
```
Comment (o-alex) [inline docs/user_guides/projects/python/environment_history.md:9]: you want to restore an older environment you can use it 
-> 
you want to restore an older environment `so` you can use it

**Review by o-alex — CHANGES_REQUESTED** [review]

File: docs/user_guides/projects/python/environment_history.md (hunk 3 above)
Comment (gibchikafa) [reply docs/user_guides/projects/python/environment_history.md:9]: I don't think it will make sense to add another `so`.

**Review by o-alex — APPROVED** [review]

**Pull request merged**